packages/styles/scss/components/buttongroup/_buttongroup.scss

```diff
@@ -34,7 +34,7 @@
   .#{$prefix}--buttongroup-item,
   :host(#{$dds-prefix}-button-group-item),
   :host(#{$dds-prefix}-button-cta) {
-    margin-top: $carbon--layout-01;
+    margin-bottom: $carbon--layout-01;
```

Comment:
@jkaeser this looks like this changes all instances of button group across all components, not sure if this was the intent?

Reply:
@jeffchew The intent here was to leverage Percy to see how many components are affected by this change, then make a decision about how to move forward. The existing styles contribute to the spacing issues reported in #5779.

Reply:
Particularly, the challenge I'm facing here relates to how two deeply nested web components each have margin rules that combine to create a gap that is too large, and the `<dds-cta-block>` component's CSS cannot reach into their shadow roots to override those margins.

@NSDrowned and I discussed, and the two options we came up with were:
1. Extend the `<dds-button-group-item>` into something like `<dds-cta-button-group-item>` so that I can control it's CSS directly.
2. Change the global styling and see how many regressions it causes.

Option 1 feels a little heavy handed, so we agreed to try option 2 first.

Reply:
Adding @guilhermelMoraes here, as @NSDrowned became a father yesterday!